content/en/events/2022/kcsna/faq.md

```diff
@@ -0,0 +1,117 @@
+
+[KubeCon North America]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Huntington Place Convention Center</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/2022/kcsna/location/
+
+
+### Will I be able to attend remotely?
+
+No, based on our experience with Kubecon NA 2021, we do not expect to be able
+to support remote access for this event.
+
+
+### Do I need to wear a mask?
+
+All attendees are required to wear masks except when eating or drinking. Eating and drinking should be kept to designated areas.
+
+### Why do I need to be a Kubernetes Org member to attend in-person?
+
+We welcome everyone to start contributing to the project, but with 1600+
+Kubernetes Org members and limited space and budget requirements we've
+made the decision to limit attendance to those that are already actively
+engaged with the project.
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at community@kubernetes.io about registering.
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
+
+
+
+[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
+
+
+### What is a sponsored attendee?
+
+If you are a SIG lead or subproject owner and there is someone that is not yet
+an org member or from another project that might take part in a panel,
+SIG face-to-face meeting etc, you can sponsor them to attend the summit.
+Please [email us] to make arrangements.
+
+### I have an idea for a session
+
+A CFP will be lauching mid August, check back soon!
+
+### My SIG wants to meet at the Summit
+
+We will provide space for SIG, WG, Committee, and other Kubernetes team meetings
+at the Summit. If you think your team will want to hold one of these, please use
+the CFP form to apply.
+
+### Will there be a doc sprint?
+
+Yes, SIG-Docs plans to hold a Doc Sprint at the Summit, which will run for the
+full day from 11am to 5pm.
+
+### Will there be a New Contributor Workshop?
+
+No, they are very time, person and resource intensive. We encourage those
+interested in becoming a contributor to use the New Contributor Course 
+slated to launch by KubeCon NA.
```

Comment:
as this is KubeCon NA, do we have any details on it, looks confusing if it's gonna launch at NA but we have nothing to link to

Reply:
No details yet, but I updated the language slightly 👍